src/broadcast.jl

```diff
@@ -0,0 +1,29 @@
+Base.Broadcast._containertype(::Type{<:DataValue}) = DataValue
+
+Base.Broadcast.promote_containertype(::Type{Any}, ::Type{DataValue}) = DataValue
+Base.Broadcast.promote_containertype(::Type{DataValue}, ::Type{Any}) = DataValue
+Base.Broadcast.promote_containertype(::Type{Tuple}, ::Type{DataValue}) = Tuple
+Base.Broadcast.promote_containertype(::Type{DataValue}, ::Type{Tuple}) = Tuple
```

Comment:
Does this play well with `Nullable`? What should broadcasting over `DataValue` and `Nullable` do? I think currently it might throw an error?